renderer/test/js_ethereum_provider_browsertest.cc

```diff
@@ -333,6 +472,25 @@ IN_PROC_BROWSER_TEST_F(JSEthereumProviderBrowserTest, NonConfigurable) {
   EXPECT_TRUE(content::EvalJs(primary_main_frame(), overwrite).ExtractBool());
 }
 
+IN_PROC_BROWSER_TEST_F(JSEthereumProviderBrowserTest,
+                       BraveEthereum_NonConfigurable) {
+  brave_wallet::SetDefaultEthereumWallet(
+      browser()->profile()->GetPrefs(),
+      brave_wallet::mojom::DefaultWallet::BraveWallet);
```

Comment:
braveEthereum should be non configurable in both prefer extension and brave wallet setting.

Reply:
Added check for both